www/src/components/landingPage/tweets/tweets.astro

```diff
@@ -1,67 +1,89 @@
 ---
 import PageSection from "../pageSection.astro";
 import TweetSlider from "./tweetSlider.astro";
-import { type Tweet, dummyTweet } from "./types";
+import type { Tweet } from "./types";
 
-const token = import.meta.env.TWITTER_BEARER_TOKEN;
-const fetchTweets = async (ids: string[]) => {
-  const url =
-    "https://api.twitter.com/2/tweets" +
-    `?ids=${ids.join(",")}` +
-    "&tweet.fields=created_at,public_metrics" +
-    "&user.fields=name,profile_image_url,username,verified" +
-    "&expansions=author_id";
+let tweets: Tweet[] = [
```

Comment:
i think this can be in a separate file along with the typedef or smth

Reply:
Good point. I moved it to the typedef file, and renamed that file as well as const `featuredTweets`.